site/content/docs/manifest/rd-web-service.ja.md

```diff
@@ -138,8 +140,17 @@ Service のインスタンスに割り当てる CPU ユニット数。指定可
 
 <div class="separator"></div>
 
-<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String</span>  
-`docker build --platform` で渡すオペレーティングシステムとアーキテクチャ。（`[os]/[arch]` の形式で指定）
+<a id="network" href="#network" class="field">`network`</a> <span class="type">Map</span>      
+`network` セクションには、Environment の VPC 内の AWS リソースにサービスを接続するためのパラメータが含まれています。Service を VPC に接続することで、[サービス検出](../developing/service-discovery.ja.md)を使用して Environment 内の他の Service と通信したり、[`storage init`](../commands/storage-init.ja.md)で Amazon Aurora などの VPC 内のデータベースに接続することができます。
+
+<span class="parent-field">network.</span><a id="network-vpc" href="#network-vpc" class="field">`vpc`</a> <span class="type">Map</span>    
+Service からの Egress トラフィックをルーティングする VPC 内のサブネットを指定します。
```

Comment:
Please be mindful of consistency in the terminology 'egress'. On l147 egress is capitalized ('Egress") while on l152 is is not capitalized ('egress').